int32 dist
---
float64 aov